job-runner/README.md

```diff
@@ -1,5 +1,96 @@
 # MedBook-JobRunner
 
+## Jobs
+
+### GSEA: preranked + leading edge (`RunGSEA`)
+
+This job runs GSEAPreranked and then GSEA's leading edge analysis to generate a heat map.
+
+#### Arguments
+
+GSEA requires a gene set and a gene set group as arguments.
+
+Optionally, the user can run the tool with a gene set created in another tool, such as the outlier analysis or Limma. These links/buttons would be found on the origin tool output page and not in the GSEA tool creation UI.
+
+Optionally, the user can provide a phenotype (two sample groups). This will produce a heat map of samples vs. enriched gene sets.
+
+- `gene_set_id`, `gene_set_name`: id and name of the gene set
+- `gene_set_sort_field`: the field on which to sort the gene set. If this is not provided, the organic sort is used. (The order with which the gene set was loaded, stored in `gene_labels`.)
+- `gene_set_group_ids`, `gene_set_group_names`: ids and names of the gene set groups. If multiple are provided, combine them into a single .gmt.
+- `gene_set_associated_object`: provided if the job is running with a gene set created by another job. This is used to link to the gene set's origin.
+- phenotype: optionally provided, produces a heat map of samples vs. pathways. NOTE: not currently implemented
+  - `sample_group_a_id`: id for sample group A
+  - `sample_group_b_id`: id for sample group B
+  - `sample_group_a_name`: name for sample group A
+  - `sample_group_b_name`: name for sample group B
+- `set_max`, `set_min`, `plot_top_x`, `nperm`, `metric`, `scoring_scheme`: arguments provided to GseaPreranked
+
+[See here for GSEA tool documentation](http://www.broadinstitute.org/cancer/software/gsea/doc/GSEAUserGuideFrame.html).
+
+#### Output
+
+GSEA's preranked tool creates a folder of HTML and data files which are linked together via relative hyperlinks. All of the files in this folder are saved as blobs. Leading Edge creates a heat map (png), which is also saved as a blob.
+
+All of these files are stored as blobs with the `associated_object` being the job object. The GESA HTML folder entry-point is named `index.html`, and the heat map is named `leading_edge_heat_map_clustered.png`.
+
+### Paired Tumor Analysis (`RunPairedAnalysis`)
```

Comment:
It's a feature that isn't ready for prime time just yet. Ted/Robert/Rob still have to discuss what exactly should happen to this code. For now let's keep it in the README until something is decided.